lib/media/media_source_engine.js

```diff
@@ -1265,6 +1265,7 @@ shaka.media.MediaSourceEngine = class {
           shaka.util.Error.Severity.CRITICAL,
           shaka.util.Error.Category.MEDIA,
           shaka.util.Error.Code.MEDIA_SOURCE_OPERATION_THREW,
+          this.video_.error || 'No error in the media element',
```

Comment:
This should be the second data item, not the first, so as to not change the order of the data items. E.g. so that people checking `error.data[0]` to find what the exception was won't suddenly be seeing the video error instead.

You should also update the documentation for this error code, in `lib/util/error.js`.

Reply:
Makes sense. Have changed it now